src/docs/Components/Accordion/code.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: 'Code'
+order: 2
+---
+
+## Import
+
+Import *the settings* and *the accordion* scss files. 
+```scss
+@import 'settings-tools/_all-settings';
+
+@import 'components/_c.accordion';
+```
+
+## Basic usage
+To create a basic accordion, wrap a `checkbox`, a `label` and a `div` inside a `div` and apply the following classes: 
```

Comment:
Can we rephrase it in a simpler way? Maybe : 

_To create a basic accordion,apply the following classes on this HTML elements:_